doc/source/changes/version_0_25.rst.inc

```diff
@@ -137,13 +138,36 @@
 Miscellaneous improvements
 --------------------------
 
-* implemented Session.to_globals(inplace=True) which will update the content of existing arrays instead of creating new
-  variables and overwriting them. This ensures the arrays have the same axes in the session than the existing variables.
-
 * added icon to display in Windows start menu and editor windows.
 
 * added 'Getting Started' section in documentation.
 
+* implemented axes argument to ipfp to specify on which axes the fitting procedure should be applied (closes
+  :issue:`185`). For example, let us assume you have a 3D array, such as:
+
+    >>> initial = ndrange('a=a0..a9;b=b0..b9;year=2000..2016')
+
+  and you want to apply a 2D fitting procedure for each value of the year axis. Previously, you had to loop on that
+  year axis explicitly and call ipfp within the loop, like:
+
+    >>> result = zeros(initial.axes)
+    >>> for year in initial.year:
+    ...     current = initial[year]
+    ...     # assume you have some targets for each year
+    ...     current_targets = [current.sum(x.a) + 1, current.sum(x.b) + 1]
+    ...     result[year] = ipfp(current_targets, current)
+
+  Now you can apply the procedure on all years at once, by telling you want to do the fitting procedure on the other
+  axes. This is a bit shorter to type, but this is also *much* faster.
+
```

Comment:
"much faster" but bellow you say that you made ipfp 10 to 20% faster. This not "much" or are you talking about 2 different things?